src/conn/conn_open.c

```diff
@@ -231,21 +231,22 @@ __wti_connection_workers(WT_SESSION_IMPL *session, const char *cfg[])
     WT_RET(__wti_tiered_storage_create(session));
     WT_RET(__wt_logmgr_create(session));
 
-#ifndef _MSC_VER
-    /*
-     * If we're performing a live restore start the server. This is intentionally placed before
-     * recovery starts as we assist recovery by copying over the log files in the background.
-     */
-    WT_RET(__wt_live_restore_server_create(session, cfg));
-#endif
-
     /*
      * Run recovery. NOTE: This call will start (and stop) eviction if recovery is required.
      * Recovery must run before the history store table is created (because recovery will update the
      * metadata, and set the maximum file id seen), and before eviction is started for real.
      */
     WT_RET(__wt_txn_recover(session, cfg));
 
+#ifndef _MSC_VER
+    /*
+     * If we're performing a live restore start the server. This is intentionally placed _after_
```

Comment:
I don't think it needs the `_` around `after`. The comment makes sense to someone new, not knowing that it used to be before recovery without any emphasis.

Reply:
24eb48fc4